### DISCUSSION TITLE
1.19 Allay Block Events

### DISCUSSION THREAD

**Discussion by bckd00r** [PaperMC/Paper#10544] [Ideas]
### Is your feature request related to a problem?

Many new mechanics and some creatures have been added to the game, and we are having some control problems with them.

### Describe the solution you'd like.

Allays interacting with note blocks spoils the whole server and our expectations, we want some trigger events for Allays.

We need to prevent them from interacting with the note blocks.

### Describe alternatives you've considered.

AllayBlockTargetEvent

AllayBlockDropItem

AllayEntityDropItem

or similar basic configuration events do not work for PathFinder Allays.

**Comment by JMBoy01** [comment]
Hello, I want to try implementing the events... I don't think any one else did so far so...
